### DISCUSSION TITLE
Add resume polling API to cancel sleep attempt to not miss drivers

### DISCUSSION THREAD

**Discussion by ramonsmits** [teslamate-org/teslamate#2587] [Ideas]
I've been using Teslafi and use it in combination with Tasker. I have it configured to do a very aggressive sleep mode that start immediately when parked at home. When I would go out my phone connects to the cars its bluetoohth and the Tasker app will invoke the Teslafi API at ` https://www.teslafi.com/feed.php?token=TOKEN&command=wake`, this will resume polling the Tesla API. This way I basically never miss out on a trip unless the car its MCU does not have any LTE connectivity (for which I also turn on my wifi hotspot).

For some reason TeslaMate sometimes also misses drives. I'm wondering if that it because at that time TeslaMate is still in a sleep attempt? Having a similar API to cancel the sleep attempt would be useful.

I see in the UI that the home screen has a "cancel sleep attempt" button but I do not see a HTTP call after clicking that. I guess that happens over the websocket connection and pressing F12 shows `resume_polling` send as a command. Maybe this can already be done via an HTTP API too.

**Comment by sjonez** [comment]
This post was a while ago so I don't know if you still need this but I am in the same situation where I want teslamate to stop attempting to sleep when my phone connects to the cars bluetooth and I discovered that you can simply send a PUT request to **/api/car/[id]/logging/resume** (or **/api/car/[id]/logging/suspend** for the opposite effect, if you want the car to instantly begin sleeping)
